k8s/demo/common/ccd/admin-web.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     nodejs:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/ccd/admin-web:prod-bf9466b3
+      image: hmctspublic.azurecr.io/ccd/admin-web:demo
```

Comment:
this won't work, the demo tag will never be updated, you need to have a PR open if you want a non prod version to be updated

Reply:
ah ok @timja will make that change